### ISSUE TITLE
Show WWT's open source status and license more clearly.

### ISSUE THREAD

**Issue by erinjohn** [WorldWideTelescope/wwt-website#15]
Add link to the MIT Open Source License on the WorldWide Telescope site and OpenWWT menu (before 11/7 developer workshop)

**Comment by kfogel** [comment]
Expanding this issue to be about explaining WWT's open source status more clearly in general, which includes stating its license.

We probably don't want to have a page named http://worldwidetelescope.org/OpenWWT/ -- the "OpenWWT" label is one we're trying to retire.  Instead, the open source nature of the project should be woven into all the pages in a context-appropriate way.  For example, the front page should state clearly that WWT is open source software, the landing page for developers should point to the GitHub repositories, and "Getting WWT" and "Licensing" pages might mention the specific open source license.

Once this is done, the references on https://en.wikipedia.org/wiki/WorldWide_Telescope should be updated accordingly.  (And in general that page needs updating; I'll file a separate issue about that, counting it as part of WWT's general web presence.)
